modules/verifying-serverless-app-deployment.adoc

```diff
@@ -6,12 +6,13 @@
 [id="verifying-serverless-app-deployment_{context}"]
 = Verifying your serverless application deployment
 
-To verify that your serverless application has been deployed successfully, you must get the application URL created by Knative, and then send a request to that URL and observe the output.
+To verify that your serverless application has been deployed successfully, you must get the application URL created by Knative, and then send a request to that URL and observe the output. {ServerlessProductName} supports the use of both HTTP and HTTPS URLs, however the output from `oc get ksvc` always prints URLs using the `http://` format.
```

Comment:
Do have context for https support? If so we can link it here?

Reply:
Well, we can add links to websites, but not xrefs to other places in our docs, those can only be included in assemblies. I don't think we have any docs on http support anyway downstream but I'm not sure.